src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,74 +4,85 @@
-
-import com.ctre.phoenix6.hardware.TalonFX;
-import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.configs.FeedbackConfigs;
+import com.ctre.phoenix6.configs.MagnetSensorConfigs;
 import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.generated.TunerConstants;
 
 public class Shooter extends SubsystemBase {
 
-  private final TalonFX pivotMotor = new TalonFX(14, Constants.OperatorConstants.canivoreSerial);
+  public final TalonFX pivotMotor = new TalonFX(14, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX leftShootMotor = new TalonFX(5, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX rightShootMotor = new TalonFX(13, Constants.OperatorConstants.canivoreSerial); 
   private double targetVelocity = 0;
   private double motorVelocity = 0;
   private boolean motorsAtShootingSpeed = false;
-  private VelocityVoltage velocity = new VelocityVoltage(targetVelocity);
-  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0.1);
+  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0);
   private final String SMART_DASHBOARD_VELOCITY = "Shooter Motor Velocity";
   private final String SMART_DASHBOARD_TARGET_VELOCITY = "Shooter Motor Target Velocity";
   private final String SMART_DASHBOARD_POSITION = "Shooter Motor Position";
+  private final String SMART_DASHBOARD_TARGET_POSITION = "Shooter Motor Target Position";
 
-  public enum Position {
-    DEFAULT,
-    AMP,
-    SPEAKER;
-  }
-
-  private static Position positionOfArm = Position.DEFAULT;
+  public CANcoder wristPositionEncoder = new CANcoder(18);
+  private double targetArmPosition = 0;
+  private static double positionOfArm = 0;
 
   public void shooterInit() {
     SmartDashboard.putNumber(SMART_DASHBOARD_VELOCITY, motorVelocity);
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
-    SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
+    SmartDashboard.putNumber(SMART_DASHBOARD_POSITION, positionOfArm);
+    SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_POSITION, targetArmPosition);
+
     leftShootMotor.setInverted(true);
+
+    MagnetSensorConfigs wristPositionMagnetConfigs = new MagnetSensorConfigs();
+    wristPositionMagnetConfigs.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+    wristPositionMagnetConfigs.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    wristPositionMagnetConfigs.MagnetOffset = 0;  // TODO Calibrate this with a command
```

Comment:
Since we made the command, I'd specify the command. When it's calibrated, just remove the `TODO` portion so people know how to get a new value if needed.